### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by SimonHerrera**
First submission for evaluation.

### CODE REVIEW

File: js/quiz.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+    //Listens for 'keyup' and replaces description on the DOM with keystrokes
+    userInput.addEventListener('keyup', function(event) {
+      activeCard.querySelector("h5").innerHTML = userInput.value;
+    });
+
+    //Listens for 'click' and changes availability text on DOM
+    availableUpdate.addEventListener('click', function(event) {
+    activeCard.querySelector("h3").innerHTML = "Car No Longer Available";
+    CarLot.setFocus();
```
Comment (NewEvolution) [inline js/quiz.js]: Lines 49 & 50 should be indented a step

File: js/quiz.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      + `<h4>Price: $${inventory[i].price}</h4>`
+      + `<h5 class="description">${inventory[i].description}</h5>`
+      + `<h3>Car Still Available: ${inventory[i].available}</h3>`
+      + `<img class='pics' src = "${inventory[i].url}"`
+      + `</article>`;
+    };
+
+    //Loop to apply the color from the inventory json description to the border
+    for(j = 0; j < initBorderColor.length; j++) {
+    initBorderColor[j].style.borderColor = `${inventory[j].color}`;
```
Comment (NewEvolution) [inline js/quiz.js]: Should be indented a step.

File: js/quiz.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  var buildCards = document.getElementById("build-cards");
+  var initBorderColor = document.getElementsByClassName("card-style");
+  var userInput = document.getElementById('user-input');
+  var availableUpdate = document.getElementById('user-button');
+
+  // Loop over the inventory and populate the page
+  loadCars.updateDOM = function(inventory) {
+    for(i = 0; i < inventory.length; i++) {
+      buildCards.innerHTML += ``
+      + `<article class="col-md-4 card-style"  id="Card-Id"  >`
```
Comment (NewEvolution) [inline js/quiz.js:12]: Good use of article 👍

File: js/carlot.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    loadCars.getCarInventory = function () {
+    return inventory;
+    };
+
+    loadCars.loadInventory = function(callback) {
+      var carSpecs = JSON.parse(this.responseText);
+      CarLot.updateDOM(carSpecs.cars);
+      inventory = carSpecs.cars;
+      console.log("test", inventory );
+    };
```
Comment (NewEvolution) [inline js/carlot.js]: Lines 5-16 should be unindented a step.

File: js/carlot.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    loadCars.getCarInventory = function () {
+    return inventory;
+    };
+
+    loadCars.loadInventory = function(callback) {
+      var carSpecs = JSON.parse(this.responseText);
+      CarLot.updateDOM(carSpecs.cars);
+      inventory = carSpecs.cars;
+      console.log("test", inventory );
```
Comment (NewEvolution) [inline js/carlot.js]: Console logs should be removed from final submitted code.

File: js/carlot.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+var CarLot = (function (loadCars) {
+
+  var inventory = [];
+
+    var inventoryLoadRequest = new XMLHttpRequest();
+
+    loadCars.getCarInventory = function () {
+    return inventory;
```
Comment (NewEvolution) [inline js/carlot.js:8]: Indent a step.

File: js/card-options.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    userInput.value = "";
+    userInput.focus();
+  };
+
+  //changes back to initial load border thickness and bgcolor
+  loadCars.resetCard = function (clickEvent) {
+    var cards = document.querySelectorAll('.card-style');
+      for (i = 0; i < cards.length; i++) {
+        cards[i].classList.remove("card-change");
+      };
```
Comment (NewEvolution) [inline js/card-options.js]: This for loop should be unindented a step, to the same level as the `var` declaration on line 17

File: js/card-options.js [hunk 8]
```diff
@@ -0,0 +1,25 @@
+var CarLot = (function (loadCars) {
+  var userInput = document.getElementById('user-input');
+
+  //Changes card border thickness and background color
+  loadCars.changeCardAttr = function (clickEvent) {
+  clickEvent.classList.add("card-change"); //add class
```
Comment (NewEvolution) [inline js/card-options.js]: Indent a step.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Car Cards</title>
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+    <link href="css/quiz.css" rel="stylesheet" >
+  </head>
+  <body>
+  <div class="container">
```
Comment (NewEvolution) [inline index.html]: Sub-tags should be indented a step below the body tag.

**Comment by NewEvolution** [comment]
All indentation issues, really.  Code itself looks & functions great.  Really clever use of the `h3` and `h5` tags to target parts of the card for editing/changing instead of some of the hoops I've seen other students have to use.

**Comment by NewEvolution** [comment]
👍

**Comment by SimonHerrera** [comment]
It's a parking lot from old hickory. I'm doing what I can.
Simon

**Pull request merged**